workshops/scrapbook_css/README.md

```diff
@@ -0,0 +1,249 @@
+
+<img src="https://cloud-5j06exp7f.vercel.app/screenshot_2020-09-12_at_7.29.11_pm.png" width="380" alt="Sam Poder's Scrapbook Profile">
+
+Today, we’re going to be learning about CSS by customizing a [Scrapbook](https://scrapbook.hackclub.com/) profile (^ like mine above). Scrapbook is a platform where hackers can share photos of what they make or just of their daily life. If you don’t have a profile, log in to the Hack Club Slack, visit the [#scrapbook](https://hackclub.slack.com/archives/C01504DCLVD/) channel, take a photo (or use an old one) and post it! 
+
+Next up, you are going to want to open up the [Scrapbook Customizer](https://scrapbook.hackclub.com/customizer/). Type in your username at the top, and delete any code in the right column. Click **‘Go’**.
+
+We’re now ready to get started!
+
+## Magical Colours!
+
+To make setting colours easier, Scrapbook uses CSS variables. CSS variables work like variables do in any other programming language. You define the variable and then instead of pasting the value in everywhere you add `var(--variable-name)`. This makes it so that if you want to change a theme colour (like we're doing today!) you can do it with one line.
+
+For Scrapbook there are 3 key colour variables that we’re going to want to play around with today. These are the background colour, the colour of text and the colour of the posts (referred to as colors-elevated below). 
+
+To tell the web browser the exact colour we want, we can use either HEX codes, or RGB values. For this workshop, we’re going to use HEX codes. To help us get those, we can [Google “colour picker”](https://www.google.com/search?q=color+picker). From here, you can play around with the slides and choose your preferred colour. Choose 3 colours to be used for the profile.
+
+Then in the right column, type:
+
+```css
+:root { 
+
+  --colors-background: YOURCOLOUR; 
+
+  --colors-text: YOURCOLOUR; 
+
+  --colors-elevated: YOURCOLOUR;
+
+}
+```
+
+Replace `YOURCOLOUR` with the HEX codes you found, make sure to include the hash (hashtag) symbol at the start.
+
+You may still see some grey, these are elements that use the `--colors-muted` variable, can you work out how to change them based on what we learnt above?
+
+I choose these colours:
+
+<img src="https://cloud-loib1whnd.vercel.app/screenshot_2020-09-12_at_8.01.02_pm.png" width="380" alt="Green and purple themed Scrapbook">
+
+## Fancy fonts ❡
+
+What if your Scrapbook was in cursive? Or in a crazy techno font! Fonts can give your page a lot more character, so let’s switch them up.
+
+ is a great place to get started, scroll through all their fonts and find one you like! Click it and then for each of its styles click “Select this Style”. Then in the bar that has popped up on the right, click `Embed` and choose `@import`. Copy the first text box without the `<style>` tags. You should have something like:
```

Comment:
Is this a mistake?

Reply:
yes, i think this said google fonts... not sure what happened will fix